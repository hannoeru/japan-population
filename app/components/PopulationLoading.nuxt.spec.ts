import { describe, expect, it } from 'vitest'

import { mount } from '@vue/test-utils'
import PopulationLoading from './PopulationLoading.vue'

describe('populationLoading', () => {
  it('renders correctly', () => {
    const wrapper = mount(PopulationLoading)
    expect(wrapper.text()).toContain('読み込み中...')
  })
  it('matches snapshot', async () => {
    const component = mount(PopulationLoading)
    expect(component.html()).toMatchSnapshot()
  })
})
