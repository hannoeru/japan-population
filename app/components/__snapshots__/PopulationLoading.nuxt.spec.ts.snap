// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`populationLoading > matches snapshot 1`] = `
"<div class="h-100 flex flex-col items-center justify-center text-gray-4"><span class="i-ph-person-arms-spread mb-3 animate-bounce text-3xl"></span>
  <p data-testid="loading-text"> 読み込み中... </p>
</div>"
`;
